tox.ini

```diff
@@ -72,3 +72,8 @@ requires =
 commands =
     pip freeze
     python setup.py build_sphinx -W
+
+[pytest]
+markers =
+    flaky: mark tests to be rerun
```

Comment:
do install pytest-rerunfailures (included in [test]), that should register this.

Reply:
This does not seem to solve the problem for me. Do I need to add something to the tox command beyond `tox -e test `?

Reply:
`tox -e test` works for me, picks up and installs rerunfailures for me. Have you tried it from a clean repo, e.g. after you removed the `.tox` directory?:

```
test create: /Users/bsipocz/munka/devel/astroquery/.tox/test
test inst: /Users/bsipocz/munka/devel/astroquery/.tox/.tmp/package/1/astroquery-0.4.7.dev8171.tar.gz
WARNING: Discarding $PYTHONPATH from environment, to override specify PYTHONPATH in 'passenv' in your configuration.
test installed: astropy==5.1.1,astroquery==0.4.7.dev8171,attrs==22.1.0,beautifulsoup4==4.11.1,certifi==2022.9.24,charset-normalizer==2.1.1,click==8.1.3,contourpy==1.0.6,coverage==6.5.0,cycler==0.11.0,exceptiongroup==1.0.0,Flask==2.2.2,fonttools==4.38.0,html5lib==1.1,hypothesis==6.56.4,idna==3.4,iniconfig==1.1.1,itsdangerous==2.1.2,jaraco.classes==3.2.3,Jinja2==3.1.2,keyring==23.9.3,kiwisolver==1.4.4,MarkupSafe==2.1.1,matplotlib==3.6.2,more-itertools==9.0.0,numpy==1.23.4,packaging==21.3,Pillow==9.3.0,pluggy==1.0.0,psutil==5.9.3,pyerfa==2.0.0.1,pyparsing==3.0.9,pytest==7.2.0,pytest-arraydiff==0.5.0,pytest-astropy==0.10.0,pytest-astropy-header==0.2.2,pytest-cov==4.0.0,pytest-dependency==0.5.1,pytest-doctestplus==0.12.1,pytest-filter-subpackage==0.1.1,pytest-mock==3.10.0,pytest-openfiles==0.5.0,pytest-remotedata==0.3.3,pytest-rerunfailures==10.2,python-dateutil==2.8.2,pyvo==1.4,PyYAML==6.0,requests==2.28.1,six==1.16.0,sortedcontainers==2.4.0,soupsieve==2.3.2.post1,tomli==2.0.1,urllib3==1.26.12,webencodings==0.5.1,Werkzeug==2.2.2
test run-test-pre: PYTHONHASHSEED='1685369747'
test run-test: commands[0] | pip freeze
```